packages/hierarchies/src/hierarchies/internal/FilteringHierarchyDefinition.ts

```diff
@@ -27,6 +34,8 @@ export interface FilteringQueryBuilderProps {
   nodeIdentifierPaths: HierarchyFilteringPath[];
 }
 
+type FilterTarget = boolean | FilterTargetGroupingNodeInfo;
```

Comment:
You separated `isFilterTarget` and `autoExpandUntil` attributes in `HierarchyNode`, but them combined here. Is there a reason?

Reply:
This greatly simplifies the amount of changes required to the logic. Also, there'd have to be 1 row added to the query in `applyECInstanceIdsFilter`.

Reply:
made the changes to remove ambiguity